metabase/metabase
# Repro #25031: Embedding Dashboard with Locked parameters does not allow numeric values

### Status
PENDING REVIEW

### What does this PR accomplish?
- Reproduces #25031

### How to test this manually?
- `yarn test-cypress-open`
- `metabase/scenarios/embedding/reproductions/20845-25031-locked-numeric-param.cy.spec.js`
- Replace `it.skip()` with `it.only()` to run the test in isolation
- The test should fail until the related issue is fixed

### Additional notes:
- Once the issue is fixed, please remove the `.skip` part (unskip the test completely)
- Make sure the test is passing and
- Merge it together with the fix

### Screenshots:
![image](https://user-images.githubusercontent.com/31325167/186938184-4fb66744-d078-4684-8b59-b1497ab3aef7.png)
